### PR TITLE
Fix #464

### LINKED ISSUE #464: video quality selection
**In what category does your enhancement fall? [choose: safety, speed, ux, qa, other]**
ux
image

sometimes animes in best video quality so much freezing and caching. video quality selection would be great here instead of restarting the script for select lower video quality

### PR DESCRIPTION
**Pull request by 71zenith**

### CODE REVIEW

**Comment by ura43** [comment]
Seems like you missed the most recent bugfixes, merge them into your fork please.
Functionality wise, could/should quality selection be a submenu?

**Comment by ura43** [comment]
can confirm working tho

File: ani-cli [hunk 1]
```diff
@@ -628,6 +629,11 @@ if [ -z "$select_first" ]; then
 			ep_choice_start=$((episode + 1))
 			ep_choice_end=""
 			;;
+        	b)
```
Comment (pystardust) [inline ani-cli:632]: try using tabs here to maintain uniform indent

**Comment by RaynardGerraldo** [comment]
can confirm working

File: ani-cli [hunk 2]
```diff
@@ -619,6 +619,7 @@ if [ -z "$select_first" ]; then
 	menu_line_alternate "replay current episode" "r"
 	menu_line_alternate "search for another anime" "a"
 	menu_line_alternate "search history" "h"
+	menu_line_alternate "select quality" "b"
```
Comment (pystardust) [inline ani-cli:622]:
```sh
menu_line_alternate "select quality (current: $quality)" "b"
```
to let the user know what the current quality is

File: ani-cli [hunk 3]
```diff
@@ -628,6 +629,11 @@ if [ -z "$select_first" ]; then
 			ep_choice_start=$((episode + 1))
 			ep_choice_end=""
 			;;
+        	b)
+			prompt "Select quality"
+			quality="$REPLY"
+			video_url=$(get_video_quality "$dpage_link")
+			;;
```
Comment (pystardust) [inline ani-cli:636]: Also why not just set `quality`
just this here works
```sh
		b)
			prompt "Select quality"
			quality="$REPLY"
			;;
```I

**Review by pystardust — CHANGES_REQUESTED** [review]

File: ani-cli (hunk 3 above)
Comment (pystardust) [reply ani-cli:636]: what the point of fetching video_url? we are anyways fetching it open_episode, the current approach does a redundant part which slows the script

Comment (71zenith) [reply ani-cli:636]: Okay, will remove that part

File: ani-cli (hunk 1 above)
Comment (71zenith) [reply ani-cli]: I was using tabs before the new commit

**Review by pystardust — APPROVED** [review]

**Pull request merged**